DOCS-5713 package update

wwarner-salesforce opened the pull request:
For ACM package update 2019-08-24, to be merged Aug 22.

https://www.mulesoft.org/jira/browse/DOCS-5713

https://www.mulesoft.org/jira/browse/AAP-1005?filter=24600 shows issues.

wwarner-salesforce committed:
AAP-1005 API Details Button

AAP-1005 API Details Button

wwarner-salesforce committed:
AAP-983 and AAP-922 Learn More Button

AAP-983 and AAP-922 Learn More Button

wwarner-salesforce committed:
AAP-912 confirm user activate and deactivate

AAP-912 confirm user activate and deactivate

wwarner-salesforce committed:
AAP-909 refresh user list

AAP-909 refresh user list

wwarner-salesforce committed:
AAP-814 API Console endpoint ordering

AAP-814 API Console endpoint ordering

wwarner-salesforce committed:
AAP-784 multiple API Consoles

AAP-784 multiple API Consoles

wwarner-salesforce commented:
I made a slight error in the commit labels: The commit labeled "AAP-909 refresh user list" includes changes for both "AAP-910 hide and show rejected users on pending users list" and "AAP-909 refresh user list".

wwarner-salesforce commented:
Remaining questions:
- In the configuration for the "API Details Button" and "Learn More Button" Lightning components, what do "API Name" and "API Version" do if there's already an "API" option? If "API" is already selected, what options are available on the "API Name" drop-down?
- Are https://www.mulesoft.org/jira/browse/AAP-922 and https://www.mulesoft.org/jira/browse/AAP-983 both about the "Learn More Button" Lightning Component?
- Do the new "API Details Button" and "Learn More Button" Lightning components change the CSS classes at https://docs.mulesoft.com/api-community-manager/css-styling ?
- Do the new "API Details Button" and "Learn More Button" Lightning components need to be added at https://docs.mulesoft.com/api-community-manager/customize as included in API Details pages?
- For AAP-910 hide and show rejected users on pending users list, when will this feature be ready to demonstrate?

jdeoliveira commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -201,6 +215,16 @@ The application listing component shows applications that use an API.
 
+== Learn More Button
+
+The learn more button enables users to access a CMS page with additional information about an API.
```
enables users to access a custom page with additional information about the API.

wwarner-salesforce replied:
Fixed. Thank you!

wwarner-salesforce committed:
Update acm-lightning-components.adoc

wwarner-salesforce committed:
update API Name and API Version description

wwarner-salesforce committed:
add CSS classes for new components

wwarner-salesforce committed:
Customize: Learn More and API Details buttons

wwarner-salesforce commented:
- "In the configuration for the "API Details Button" and "Learn More Button" Lightning components, what do "API Name" and "API Version" do if there's already an "API" option? If "API" is already selected, what options are available on the "API Name" drop-down?" Emiliano Spezzirri explained that the options for "API Name" and "API Version" are "Append", "Prepend", and "None". Updated text to reflect this.
- "Are https://www.mulesoft.org/jira/browse/AAP-922 and https://www.mulesoft.org/jira/browse/AAP-983 both about the "Learn More Button" Lightning Component?" Yes, confirmed by Emiliano.
- "Do the new "API Details Button" and "Learn More Button" Lightning components change the CSS classes at https://docs.mulesoft.com/api-community-manager/css-styling ?" Yes, and Emiliano sent the new classes.
- "Do the new "API Details Button" and "Learn More Button" Lightning components need to be added at https://docs.mulesoft.com/api-community-manager/customize as included in API Details pages?" Yes, Emiliano said "Learn More" on API Details page, and "API Details" on Home page. Updated text to add these.
- "For AAP-910 hide and show rejected users on pending users list, when will this feature be ready to demonstrate?" Francisco will send screenshots.

New classes:

For Learn more button: .learn-more-button: Learn More button

For API Details Button: .api-details-button: API Details Button

wwarner-salesforce committed:
hide/show rejected requests

wwarner-salesforce commented:
Francisco sent screenshots. Updated description for hide/show rejected requests.

paigehargrave commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -56,6 +56,10 @@ The API carousel component shows a horizontally scrollable set of API cards.
 
 The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
 
+Endpoints are displayed in alphabetical order, even when the original API specification listed them in a different order.
```
Does the latter half of the sentence add value? Can we remove it and have 'Endpoints are displayed in alphabetical order.'?

wwarner-salesforce replied:
Yes, that is better. Removed the latter half.

paigehargrave commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -56,6 +56,10 @@ The API carousel component shows a horizontally scrollable set of API cards.
 
+Endpoints are displayed in alphabetical order, even when the original API specification listed them in a different order.
+
+The console components are the *API Console*, *API Console Documentation*, *API Console Instances*, *API Console Navigation*, and *API Console Request Panel*. In the Builder you can add multiple console components to the same page. If you do this, any change in one console component will also be visible in the other console components on the same page that show the same API. For example, selectin ...
```
Minor suggestions:
Console components include *API Console*, *API Console Documentation*, *API Console Instances*, *API Console Navigation*, and *API Console Request Panel*. In the Builder, you can add multiple console components to the same page. Changes to a console component are visible to all other console components on that page that show the same API. For example, selecting the *Summary* tab in an API Console results in  *Summary* tab selection in all API Consoles on the same page that show the same API.

paigehargrave replied:
Can 'display' be used vs 'show'?

wwarner-salesforce replied:
Yes, 'display' is good. Made all of these changes.

paigehargrave commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -159,6 +163,16 @@ The API console request panel component shows a panel where users can test an AP
 
+== API Details Button
+
+The API details button enables users to access details about an API.
```
Select *API Details* to specify button characteristics. (Avoid passive voice.) Is this correct? It looked like the lead sentence didn't cover the fact that the user can specify what the button looks like.

wwarner-salesforce replied:
The ACM user (reader of this doc) configures the button with the options described here and adds it to the ACM portal. Then the consumer developers using the portal can click the button to access details about an API. Clarified as "The API details button component provides portal users with details about an API."

paigehargrave commented on modules/ROOT/pages/customize.adoc:
```diff
@@ -34,6 +34,7 @@ is always present and is always the first item. The following properties can be
 ** *Publicly available*
 * https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
 * *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
+* *API Details Button*: Enables users to access details about an API.
```
Provides details about the API.

wwarner-salesforce replied:
Much better! Fixed.

paigehargrave commented on modules/ROOT/pages/customize.adoc:
```diff
@@ -44,6 +45,7 @@ Components that are typically used in *API details* pages include:
 *API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
 discussions, and questions.
 * *API Access Requestor*: Registers a client application and requests access to the API.
+* *Learn More Button*: Enables users to access a custom page with additional information about the API.
```
Provides additional information about the API.

wwarner-salesforce replied:
Fixed.

paigehargrave commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -159,6 +163,16 @@ The API console request panel component shows a panel where users can test an AP
+* *Button Label*: Set the button label. Default is `API Details`.
+* *Button Align*: Set button alignment to left, right, or center.
+* *API Name*: Choose "Append", "Prepend", or "None" to add the API name to the button label.
+* *API Version*: Choose "Append", "Prepend", or "None" to add the API version to the button label.
```
A few minor suggestions:
*API*: Choose the API to show. (Same comment as above - 'show' vs 'display')
* *Button Label*: Specifies the button label. Default is `API Details`.
* *Button Align*: Sets button alignment to left, right, or center. (Of what? What will this be next to in the UI?)
* *API Name*: Adds the API name to *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?
* *API Version*: Adds the API version to the *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?

Also, since the parameter values are UI elements, should they be bolded instead of in quotes?

wwarner-salesforce replied:
Good catch, those should be bold. Updated with these changes.

paigehargrave commented on modules/ROOT/pages/acm-lightning-components.adoc:
```diff
@@ -201,6 +215,16 @@ The application listing component shows applications that use an API.
+* *Button Label*: Set the button label. Default is `Learn more about this API`.
+* *Button Align*: Set button alignment to left, right, or center.
+* *API Name*: Choose "Append", "Prepend", or "None" to add the API name to the button label.
+* *API Version*: Choose "Append", "Prepend", or "None" to add the API version to the button label.
```
A few suggestions:
Select *Learn More* to specify button characteristics. (Avoid passive voice.) Is this correct? It looked like the lead sentence didn't cover the fact that the user can specify what the button looks like.

 * *API*: Choose the API to show. (same comment as above)
* *Button Label*: Specifies the button label. Default is `Learn more about this API`.
* *Button Align*: Sets button alignment to left, right, or center. (Of what? What will this be next to in the UI?)
* *API Name*: Adds the API name to *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?
* *API Version*: Adds the API version to the *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?

Also, since the parameter values are UI elements, should they be bolded instead of in quotes?

wwarner-salesforce replied:
Updated with these changes.

paigehargrave commented on modules/ROOT/pages/manage-members.adoc:
```diff
@@ -25,13 +25,16 @@ Consumer developers follow these steps to self-register:
-. (Optional) To deactivate a member, disable the *Active* setting for that member.
+. (Optional) To deactivate a member, disable the *Active* setting for that member and confirm the change.
+
+Select the options menu to choose how many registration requests to show on each page, and to *Show rejected users* or *Hide rejected users*.
```
Select the options menu to specify the number of registration requests to display on each page, and to *Show rejected users* or *Hide rejected users*.

wwarner-salesforce replied:
Much better! Fixed.

wwarner-salesforce committed:
Update manage-members.adoc

wwarner-salesforce committed:
Update acm-lightning-components.adoc

wwarner-salesforce committed:
Update customize.adoc
